### DISCUSSION TITLE
Maps with a hyphen (-) in its name does not show the map image

### DISCUSSION THREAD

**Discussion by RememberShephard** [tltneon/lgsl#164] [General]
I've noticed that certain maps that have the hyphen character in their name do not pull the image from the images folder, it defaults to the "No Image" image. Is this something that you have come across before?

**Comment by BaxAndrei** [comment]
Do you have any example? Also are you sure image exists?

**Comment by TacTicTow** [comment]
The lgsl script uses a string replace command and changes all hyphens to an underscore. So...

my-cool-map.jpg should be named my_cool_map.jpg
